README.rst

```diff
@@ -425,6 +428,135 @@ view (only when using HTTP POST).
 The signal is emitted just before a successful response is returned,
 it is not sent if the response was not successful.
 
+Default Metrics
+---------------
+
+``Ping``
+~~~~~~~~
+
++--------------+-------------------------------------------------+
+|     name     |              ``Ping <Device>``                  |
++--------------+-------------------------------------------------+
+|     key      |                   ``ping``                      |
++--------------+-------------------------------------------------+
+|  value type  |  ``int`` (reachable / loss), ``float`` (rtt)    |
++--------------+-------------------------------------------------+
+|  field_name  |                 ``reachable``                   |
++--------------+-------------------------------------------------+
+|configuration |                   ``ping``                      |
++--------------+-------------------------------------------------+
+|related_fields| ``loss``, ``rtt_min``, ``rtt_max``, ``rtt_avg`` |
```

Comment:
I'd merge `field_name` and `related_fields` into just `fields` in all tables, otherwise it sounds too technical and tied to the current implementation.

Keep in mind that the idea is that some newcomer can understand at first sight what are the metrics stored and what fields are available for them.

Reply:
Makes sense, it is better for a newcomer but might add slight confusion for a contributor who might not find similarity between the model and :point_up: the table. How's this, merge `field_name` and `related_fields` calling it simply `fields` but adding a small note at the end of this section mentioning the same. It should work for both, I think :)

Reply:
Yes but we are probably going to change the current implementation, and the contributor can check out and study the code at any time.